libutils/include/utils/compile_time.h

```diff
@@ -8,4 +8,10 @@
 
 #include <assert.h>
 
+#if !defined(static_assert) && \
```

Comment:
Let's add a comment here `/* 201112L indicated c11 */`. Might be extra verbose, but it's a nice reference to demystify the numer.

Reply:
Sure, will add.